dask/dataframe/core.py

```diff
@@ -575,25 +575,10 @@ def _aca_agg(self, token, func, aggfunc=None, **kwargs):
         """ Wrapper for aggregations """
         raise NotImplementedError
 
-    def rolling(self, window, min_periods=None, win_type=None, axis=0):
-        """Provides rolling transformations.
-
-        Parameters
-        ----------
-
-        window: int
-            Size of the moving window. This window must be smaller
-            than the size of the previous partition.
-        min_periods: int, default None
-            Minimum number of observations in window required to have
-            a value (otherwise result is NA).
-        win_type: string, default None
-            Provide a window type. (Identical to pandas.)
-        axis: int, default 0
-            Provide the axis to apply the function. (Identical to pandas.)
-
-        The center argument and deprecated freq argument are not supported.
-        """
+    def rolling(self, window, min_periods=None, freq=None, center=False,
+                win_type=None, axis=0):
+        # TODO: docstring. (Should soon be identical to pandas, except
+        # with a proviso about window too big for a partition.
```

Comment:
This probably needs to be resolved one way or the other.  I'm currently in favor of keeping our docstring so that we can add provisos like you suggest.